.circleci/config.yml

```diff
@@ -925,6 +957,13 @@ workflows:
               only:
                 - /^test-[\d|.]+$/
                 - /^release-[\d|.]+$/
+      - docker-update-base-images:
+          context: docker
+          filters:
+            branches:
+              only:
+                - /^test-[\d|.]+$/
+                - /^release-[\d|.]+$/
```

Comment:
Should we also run this as nightly scheduler and on branches (test-*, release-* and master)?  When we change the base image in master/trunk, I supposed we want it cached in ECR too.

Reply:
moving to master so it works with your pr.    it will only build on change to docker files that build on top of debian.   Without any way to reliably know what already been processed in the past, figured this was the least error prone approach and should still happen less frequently than one build a day.